### PR TITLE
Fix Vivillion questline not started when it should be

### LINKED ISSUE #2308: [BUG] vivillon quest
**Info**
 - OS: Windows 8
 - Browser Firefox
 - Version 0.9.5

**Description:**
The quest doesn’t start even after beating the snowbelle city gym

**Steps to Reproduce:**
1.beat snowbelle city gym in kalos
2.check if the quest begin

**Expected Behavior:**
Beginning of the quest

**Additional Information:**
Vanilla game

### PR DESCRIPTION
**Pull request by RedSparr0w**
This PR should fix where someone should have the Vivillion questline started but doesn't.

Don't have a save to test it yet though.

Fixes #2308

### CODE REVIEW

File: src/scripts/Game.ts [hunk 1]
```diff
@@ -185,6 +185,17 @@ class Game {
                 App.game.quests.getQuestLine('Mining Expedition').beginQuest(App.game.quests.getQuestLine('Mining Expedition').curQuest());
             }
         }
+        // Vivillon questline (if not started due to gym bug)
+        if (App.game.quests.getQuestLine('The Great Vivillon Hunt!').state() == QuestLineState.inactive) {
+            if (App.game.party.alreadyCaughtPokemon(666.01)) {
+                // Has obtained Aerodactyl
```
Comment (Aegyo) [inline src/scripts/Game.ts:191]: ...Aerodactyl?

**Review by Aegyo — APPROVED** [review]
Tested with the save provided in linked issue :+1:

File: src/scripts/Game.ts (hunk 1 above)
Comment (RedSparr0w) [reply src/scripts/Game.ts:193]: 👍 
```suggestion
                // Has obtained Vivillon (Pokéball)
```

File: src/scripts/Game.ts [hunk 2]
```diff
@@ -187,6 +187,17 @@ class Game {
                 App.game.quests.getQuestLine('Mining Expedition').beginQuest(App.game.quests.getQuestLine('Mining Expedition').curQuest());
             }
         }
+        // Vivillon questline (if not started due to gym bug)
+        if (App.game.quests.getQuestLine('The Great Vivillon Hunt!').state() == QuestLineState.inactive) {
+            if (App.game.party.alreadyCaughtPokemon(666.01)) {
+                // Has obtained Vivillon (Pokéball)
+                App.game.quests.getQuestLine('The Great Vivillon Hunt!').state(QuestLineState.ended);
+            } else if (App.game.badgeCase.badgeList[BadgeEnums.Iceberg]()) {
+                // Has the soul badge, Quest is started
```
Comment (RedSparr0w) [inline src/scripts/Game.ts:196]:
```suggestion
                // Has the Iceberg badge, Quest is started
```

**Pull request merged**